src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -159,7 +159,7 @@ export class PeriodicReportRepository extends DtoRepository(IPeriodicReport) {
     return await this.db
       .query()
       .match([
-        node('', '', { id: parentId }),
+        node('', 'BaseNode', { id: parentId }),
```

Comment:
It's weird that this fails when `undefined` wouldn't.

Reply:
I guess `[]` would work too. 
https://github.com/jamesfer/cypher-query-builder/blob/3986fea0a4d97eadff829de0e2fe66665529ab4f/src/utils.ts#L90